client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	isAutoRenewDisabled,
+	isIncludedWithPlan,
+	isPaidWithCreditCard,
+	isPaidWithCredits,
+	isPaidWithPayPalDirect,
+	paymentLogoType,
+	hasPaymentMethod,
+} from 'calypso/lib/purchases';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import PaymentLogo from 'calypso/components/payment-logo';
+import type { Purchase } from 'calypso/lib/purchases/types';
+
+export default function PaymentInfoBlock( { purchase }: { purchase: Purchase } ): JSX.Element {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+
+	if ( isIncludedWithPlan( purchase ) ) {
+		return <PaymentInfoBlockWrapper>{ translate( 'Included with plan' ) }</PaymentInfoBlockWrapper>;
+	}
+
+	if ( hasPaymentMethod( purchase ) ) {
+		if ( isPaidWithCredits( purchase ) ) {
+			return <PaymentInfoBlockWrapper>{ translate( 'Credits' ) }</PaymentInfoBlockWrapper>;
+		}
+
+		const logoType = paymentLogoType( purchase );
+
+		if ( isPaidWithCreditCard( purchase ) ) {
+			return (
+				<PaymentInfoBlockWrapper>
+					<PaymentLogo type={ logoType } disabled={ isAutoRenewDisabled( purchase ) } />
```

Comment:
I believe all these should be `isExpiring()` rather than `isAutoRenewDisabled()`.  Otherwise the styling will be wrong when the subscription is close to expiration.

There shouldn't actually be a need to introduce `isAutoRenewDisabled()` in this pull request, since it's covered as part of `isExpiring()`.

Reply:
Ah, _now_ I understand what you were saying above about using `isExpiring`. I was very confused because I thought it was talking about the credit card.